packages/fxa-auth-server/lib/log.js

```diff
@@ -17,9 +17,19 @@ const Sentry = require('@sentry/node');
 const ISSUER = config.get('domain') || '';
 const CLIENT_ID_TO_SERVICE_NAMES = config.get('oauth.clientIds') || {};
 
+const _registered = {};
+
 function Lug(options) {
+  const name = options.name || 'fxa-auth-server';
+
+  if (_registered[name] != null) {
```

Comment:
I'm always confused by non-strict equals... Should this be a stricter `!== undefined` to avoid possible coercion?

```sh
> const foo = {}
> foo.bar = "dan" # 'dan'
> foo # { bar: 'dan' }
> foo.baz == null # true
> foo.baz == undefined # true
> foo.baz === null # false
> foo.baz === undefined # true
```

Or conversely, a `Set()`, but that might be too heavy:

```sh
> const bar = new Set()
> bar.add("foo") # Set(1) { 'foo' }
> bar.has("foo") # true
> bar.has("baz") # false
```

Reply:
The check is logically sound and I don't feel as though coercion is a problem. As your cases indicate `x != null` is equivalent to `x !== null || x !== undefined` and IMO should be seen as idiomatic JavaScript.

I do agree that in many cases a strict equality check is preferable... Since the intent is to flag with `true`, I actually think `if(_registered[name] === true)`, is the most expressive way to write this, so I think I'll switch it to that.